src/library/scala/util/Either.scala

```diff
@@ -535,7 +533,6 @@ object Either {
      *
      *  @throws java.util.NoSuchElementException if the projection is [[scala.util.Right]]
      */
-    @deprecated("use `Either.swap.getOrElse` instead", "2.13.0")
```

Comment:
Should this stay deprecated? I un-deprecated it thinking to re-enable everything to do with `Left`, but as @julienrf pointed out, I might have been overzealous.

It seems the point of deprecating `get` here is not to get rid of left projections, but of unsafe behaviours. If so, then it probably needs to stay deprecated.

Reply:
I thought that `get` was available on `Either` (like it is available on `Option`, although being partially defined only), but it is not. Therefore I suggest to either keep it deprecated here or to make it available on `Either` as well.

Reply:
Relevant discussion in #7121

Reply:
Done in c66d6572b7.